validation/virustotal.go

```diff
@@ -78,7 +78,7 @@ func ScanFiles(ctx context.Context, files []io.Reader, names []string) ([]ScanRe
 		_ = errs.Wait()
 		close(c)
 	}()
-	var results []ScanResult
+	results := make([]ScanResult, 0, 100)
```

Comment:
Why 100?

Reply:
I think that just initializing a 0-size should be enough, setting a cap will cause problems when someone tries to upload a mod with 101 scanable items

Reply:
The linter is probably detecting that the channel will pass at most `len(fileCount)` items, and wants you to use that as the initial size, `make([]ScanResult, len(fileCount))`

Reply:
Actually, this should probably be rewritten as a `select` with a loop, because:

1. We are just ignoring if the `errs` returns an error
2. We are not guaranteed that `fileCount` many items will be sent to `c`